pkgs/development/python-modules/pyotb/default.nix

```
@@ -0,0 +1,100 @@
+{
+  lib,
+  pkgs,
+  fetchFromGitHub,
+  buildPythonPackage,
+  python,
+
+  # build-system
+  setuptools,
+
+  # dependencies
+  otb,
+
+  # tests
+  pytestCheckHook,
+  pytest-cov-stub,
+  requests,
+  writableTmpDirAsHomeHook,
+}:
+let
+  # fetch the test data separately or else none of the test will work
+  # https://github.com/orfeotoolbox/pyotb/blob/develop/tests/tests_data.py
+  spotImage = pkgs.fetchurl {
```

```suggestion
  spotImage = fetchurl {
```
And directly add `fetchurl` to the inputs.